packages/feedback/src/types/index.ts

```diff
@@ -27,3 +27,193 @@ export interface SendFeedbackData {
     name?: string;
   };
 }
+
+export interface FeedbackFormData {
+  message: string;
+  email?: string;
+  name?: string;
+}
+
+export interface FeedbackConfigurationWithDefaults {
+  /**
+   * id to use for the main widget container (this will host the shadow DOM)
+   */
+  id: string;
+
+  /**
+   * DOM Selector to attach click listener to, for opening Feedback dialog.
+   */
+  attachTo: Node | string | null;
+
+  /**
+   * Auto-inject default Feedback actor button to the DOM when integration is
+   * added.
+   */
+  autoInject: boolean;
+
+  /**
+   * If true, will not collect user data (email/name).
+   */
+  isAnonymous: boolean;
+
+  /**
+   * Should the email field be required?
+   */
+  isEmailRequired: boolean;
+
+  /**
+   * Should the name field be required?
+   */
+  isNameRequired: boolean;
+
+  /**
+   * Should the email input field be visible? Note: email will still be collected if set via `Sentry.setUser()`
+   */
+  showEmail: boolean;
+
+  /**
+   * Should the name input field be visible? Note: name will still be collected if set via `Sentry.setUser()`
+   */
+  showName: boolean;
+
+  /**
+   * Fill in email/name input fields with Sentry user context if it exists.
+   * The value of the email/name keys represent the properties of your user context.
+   */
+  useSentryUser: {
+    email: string;
+    name: string;
+  };
+
+  // * Color theme customization * //
+  /**
+   * The colorscheme to use. "system" will choose the scheme based on the user's system settings
+   */
+  colorScheme: 'system' | 'light' | 'dark';
+
+  /**
+   * Theme customization, will be merged with default theme values.
+   */
+  theme: {
```

Comment:
How does theme & colorScheme interact? This seems a bit similar to me, conceptually..?

Reply:
ah, wait, so `colorScheme` selects if we want to show the dark or light theme, and `theme` allows to customize the default light/dark theme...?